dandi/_version.py

```diff
@@ -504,7 +504,7 @@ def get_versions():
         # versionfile_source is the relative path from the top of the source
         # tree (where the .git directory might live) to this file. Invert
         # this to find the root from __file__.
-        for i in cfg.versionfile_source.split('/'):
+        for _ in cfg.versionfile_source.split('/'):
```

Comment:
this is versioneer one, can we just skip it for LGTM?

meanwhile checked -- current master in versioneer already has this:
```
$> git grep -A1 'in cfg.versionfile_source.split'
src/git/long_get_versions.py:        for _ in cfg.versionfile_source.split('/'):
src/git/long_get_versions.py-            root = os.path.dirname(root)
```
so in principle we could just proceed, but still I think we should just skip this file

Reply:
I don't know how ignoring alerts works in LGTM, so I can't answer that.

Reply:
google has all the answers: https://lgtm.com/help/lgtm/customizing-file-classification